rigex-front-end/src/components/auth/signUpForm.js

```diff
@@ -0,0 +1,185 @@
+import React, { useState, useEffect } from 'react';
+import { useMutation } from '@apollo/client';
+import { useDispatch } from 'react-redux';
+
+import Button from '@material-ui/core/Button';
```

Comment:
no need to use material ui for the button

Reply:
fixed